src/components/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+// booksSlice.js
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState: [],
+  reducers: {
+    bookAdded(state, action) {
+      state.push(action.payload);
+    },
+    bookRemoved(state, action) {
+      const index = state.findIndex((book) => book.id === action.payload);
+      state.splice(index, 1);
+    },
```

Comment:
- [ ] if the action.type is different, it should returns the state with no changes, kindly add this to the reducer

Reply:
Implemented 👍